_includes/usagepolicy.html

```diff
@@ -4,23 +4,23 @@
       <p>This page contains the policy for the Red Hat sponsored anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
       <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
       <H3>What information does Red Hat collect?</H3>
-      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat. Metrics include the following information:</p>
```

Comment:
This is not syntactically correct so it needs a fix, be it this one or something else.